src/alchemtest/amber/access.py

```diff
@@ -106,3 +106,19 @@ def load_invalidfiles():
 
     return Bunch(data=data,
                  DESCR=fdescr)
+
+
+def load_single_dHdl_file():
+    """Load a single dHdl file."""
+
+    module_path = dirname(__file__)
+
+    return glob(join(module_path, 'simplesolvated/charge/0.00/ti-*.out'))
```

Comment:
You can achieve loading a single file with
```python
import alchemtest.amber
solvated = alchemtest.amber.load_simplesolvated()

single_dHdl = solvated.data['charge'][0]
```

Reply:
Thank you, I completely missed that, I fixed it!